pkgs/applications/networking/ids/zeek/broker/caf-incubator.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, cmake
+, caf
+, openssl
+}:
+stdenv.mkDerivation {
+  pname = "caf-incubator";
+  version = "2022-09-11-zeek";
```

Comment:
```suggestion
  version = "unstable-2022-09-11";
```

Reply:
This file is a leftover from an earlier version of this PR. I'll remove.